dashboard/app/models/pd/international_optin.rb

```diff
@@ -0,0 +1,101 @@
+# == Schema Information
+#
+# Table name: pd_international_optins
+#
+#  id         :integer          not null, primary key
+#  user_id    :integer
+#  form_data  :text(65535)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_pd_international_optins_on_user_id  (user_id)
+#
+
+require 'international_optin_people'
+
+class Pd::InternationalOptin < ApplicationRecord
+  include Pd::Form
+
+  belongs_to :user
```

Comment:
Not seeing an associated `user_id` in the database after submitting this form, even when signed in.  Is there something else needed to make the association work?

Reply:
The association is fine, but its value is not being set. It should be set on initialization, [here](https://github.com/code-dot-org/code-dot-org/pull/23235/files#diff-14445ae63d23d921b409ef229f562148R3).

Reply:
Is `user_id` required? If so we should also probably update the db column to not null, add a required validation, and controller auth.